stepper.md

```diff
@@ -35,30 +36,168 @@ motor drivers). Additional delay values can be added in the future.*
+
+Query
+```c
+ /* -----------------------------------------------------
+ * 0 START_SYSEX                (0xF0)
+ * 1 Stepper Command            (0x72)
+ * 2 Stepper Get Position       (0x02)
+ * 3 Stepper #                  ([0 - MAX_STEPPERS-1])
+ * 4 END_SYSEX                  (0xF7)
+ * -----------------------------------------------------
+ */
+```
+ 
+Response 
+```c
+ /* -----------------------------------------------------
+ * 0 START_SYSEX                (0xF0)
+ * 1 Stepper Command      		(0x72)
+ * 2 Stepper Get Position       (0x02)
+ * 3 Stepper #                  ([0 - MAX_STEPPERS-1])
+ * 4 current position, bits 0-6
+ * 5 current position, bits 7-13
+ * 6 current position, bits 14-20
+ * 7 current position, bits 21-27
+ * 8 the signed value			(negative 0x00 or positive 0x01)
+ * 9 END_SYSEX                  (0xF7)
+ * -----------------------------------------------------
+ */
 ```
 
-Stepper step
+###Report Steppers's Distance from target
+
+Query
+```c
+ /* -----------------------------------------------------
+ * 0 START_SYSEX                (0xF0)
+ * 1 Stepper Command            (0x72)
+ * 2 Stepper Get Distance To    (0x03)
+ * 3 Stepper #                  ([0 - MAX_STEPPERS-1])
+ * 4 END_SYSEX                  (0xF7)
+ * -----------------------------------------------------
+ */
 ```
-0  START_SYSEX          (0xF0)
-1  Stepper Command      (0x72)
-2  config command       (0x01)
-3  device number        (0-5)
-4  direction            (0-1) (0x00 = CW, 0x01 = CCW)
-5  num steps byte1 LSB
-6  num steps byte2
-7  num steps byte3 MSB  (21 bits (2,097,151 steps max))
-8  speed LSB            (steps in 0.01*rad/sec  (2050 = 20.50 rad/sec))
-9  speed MSB
-10 [optional] accel LSB (acceleration in 0.01*rad/sec^2 (1000 = 10.0 rad/sec^2))
-11 [optional] accel MSB
-12 [optional] decel LSB (deceleration in 0.01*rad/sec^2)
-13 [optional] decel MSB
-14 END_SYSEX            (0xF7)
+ 
+Response 
+```c
+ /* -----------------------------------------------------
+ * 0 START_SYSEX                (0xF0)
+ * 1 Stepper Command      		(0x72)
+ * 2 Stepper Get Distance To    (0x03)
+ * 3 Stepper #                  ([0 - MAX_STEPPERS-1])
+ * 4 current position, bits 0-6
+ * 5 current position, bits 7-13
+ * 6 current position, bits 14-20
+ * 7 current position, bits 21-27
+ * 8 the signed value			(negative 0x00 or positive 0x01)
+ * 9 END_SYSEX                  (0xF7)
+ * -----------------------------------------------------
+ */
+```
+
+###Stepper set speed
+```c
+ /* -----------------------------------------------------
+ * 0 START_SYSEX                (0xF0)
+ * 1 Stepper Command      		(0x72)
+ * 2 Stepper set Speed 		   	(0x04)
+ * 3 Stepper #                 	([0 - MAX_STEPPERS-1])
+ * 4 Speed bits 0-6            	(least significant byte)
+ * 5 Speed bits 7-13           	(most significant byte)
+ * 6 END_SYSEX                	(0xF7)
+ * -----------------------------------------------------
+ */
+```
+
+###Stepper set acceleration
```

Comment:
There is currently no way to change this value in the FirmataStepper implementation this once a step sequence has begun.